Donation-platform/src/main/java/com/google/charities/AddCharitiesFromJSON.java

```diff
@@ -85,6 +129,51 @@
           put(RESEARCH, 20);
         }
     };
+
+  // HARDCODED MAPPING OF CAUSES TO TRENDING SCORES
+  public static final Map<String, Integer> causeScores =
+    new HashMap<String, Integer>() {
+        {
+          put(ADV, 15);
```

Comment:
why dont we inititalize this map with 0? Are we updating this score later based on google trends API?

Reply:
Done. Set to zero.